rootfs/templates/wrapper

```diff
@@ -68,19 +68,23 @@ function use() {
 					--env USER_ID
 					--env GROUP_ID)
 			fi
+			if [ "${OS}" == 'Darwin' ]; then
+				# Bind-mount SSH-agent socket (available in docker-for mac Edge release)
+				DOCKER_ARGS+=(--volume /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock
+					-e SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock)
```

Comment:
I do not like that this sets SSH_AUTH_SOCK even when `/run/host-services/ssh-auth.sock` does not exist, or that it mounts `/run/host-services/ssh-auth.sock` rather than the current value of `SSH_AUTH_SOCK`. What's wrong with following the pattern we use for Linux hosts?

Reply:
The current pattern doesn’t work on Mac. He has solving it so it works on Docker for Mac .